pyccel/stdlib/gFTL_functions/Set_extensions.inc

```diff
@@ -41,4 +41,33 @@ contains
 
   end subroutine __IDENTITY(Set)_intersection_update
 
+   function __IDENTITY(Set)_min(my_set) result(result)
+    class(Set), intent(in) :: my_set
+    __T_declare_dummy__ :: result
+
+    type(SetIterator) :: iter
+    type(SetIterator) :: last
+
+    iter = my_set%begin()
+    last = my_set%end()
+
+    result = iter%of()
```

Comment:
Here you rely on the fact that sets are sorted? Can you add a comment to indicate this

Reply:
fixed!